### DISCUSSION TITLE
Add Nessie

### DISCUSSION THREAD

**Discussion by sergkoudi** [TOSIT-IO/TDP#26] [New components] [answered]
### Presentation of the project

The transactional catalog for Data Lakes with Git-like capabilities and cross-table transactions. It provides visibility, isolation, and versioning to Data Lakes so that different environments can use the same Data Lake without risking the consistent state of production data.

It works with the Iceberg and Delta Lake table formats and extends the tables to provide a single serial view of transaction history.

Website - projectnessie.org. Apache-licensed.

### Why should it be included in TDP?

It provides useful features to data lakes on top of the trendious table formats. It could be added as an extra component for TDP to https://github.com/TOSIT-IO/tdp-collection-extras.

**Comment by mehdibn** [comment, marked as answer]
I think that Nessie is a new project and it lacks of the community support.
It proposes very advanced features that are not always required in a big data context.
Maybe, we can add it in the roadmap for future integration in TDP once the  stack is fully integrated.
